src/app/search/search.module.ts

```diff
@@ -5,14 +5,16 @@ import { FormsModule } from '@angular/forms';
 import { SearchComponent } from './search.component';
 import { WorkbasketModule } from '../workbasket/workbasket.module';
 import { SharedModule } from '../shared/shared.module';
+import { SearchFiltersModule } from '@hmcts/ccd-case-ui-toolkit';
 
 @NgModule({
   imports: [
     CommonModule,
     RouterModule,
     FormsModule,
     SharedModule,
-    WorkbasketModule
+    WorkbasketModule,
+    SearchFiltersModule,
```

Comment:
The SearchFiltersComponent is now imported via SearchFitlersModule rather than SharedModule. This is the only module that needs this component.